tests/test_apCorrMap.py

```diff
@@ -109,7 +109,7 @@ def testPersistence(self):
 
     def testExposurePersistence(self):
         """Test that the ApCorrMap is saved with an Exposure"""
-        filename = "testApCorrMap.fits"
+        filename = "testApCorrMap-exposure.fits"
```

Comment:
I suggest using:

    with lsst.utils.tests.getTempFilePath(".fits") as filename:

That gets you automatic cleanup, and is more immune to cargo culting because it chooses a name based on the test.

Ditto in `testExposureRecordPersistence`.